modal/cli/entry_point.py

```diff
@@ -45,12 +45,17 @@ def version_callback(value: bool):
 )
 
 
-@entrypoint_cli_typer.callback()
+@entrypoint_cli_typer.callback(invoke_without_command=True)
 def modal(
     ctx: typer.Context,
     version: bool = typer.Option(None, "--version", callback=version_callback),
 ):
-    pass
+    # TODO: When https://github.com/fastapi/typer/pull/1240 gets shipped, then
+    # - set invoke_without_command=False in the callback decorator
+    # - set no_args_is_help=True in entrypoint_cli_typer
+    if ctx.invoked_subcommand is None:
+        print(ctx.get_help())
```

Comment:
Is `print` sufficient to get the Rich mark up we normally see here?

Reply:
It looks okay, but I moved it to `console.print` to be safe.